python/py-tomli/Portfile

```diff
@@ -20,7 +20,7 @@ checksums           rmd160  f7f7431f41a5cc923266cf63cbc1e6bde646a2b0 \
                     size    17392
 
 # Do not add additional versions, nothing should be using this in Python 3.11 or newer.
-python.versions     36 37 38 39 310 311
+python.versions     39 310 311 312 313
```

Comment:
36/37/38 are still needed (part of build infrastructure for older versions). And I don't see why 312 and 313 are needed? Even 311 shouldn't be needed, but there are some ports that still depend on it.

See the comment right above this line!

Reply:
Whoops, I incorrectly added this commit. I'm going to drop it.